### PR TITLE
Feature/locale

### PR DESCRIPTION
**Pull request by hserang**
## Purpose
Add locale support and language template files to Basic App

## Status
Ready pending review

### CODE REVIEW

File: app/scripts/modules/payments/components/payment-create.jsx [hunk 1]
```diff
@@ -195,23 +201,48 @@ var PaymentCreate = React.createClass({
     this.props.model.off('invalid validationComplete sync error addressProcessed');
   },
 
-  render: function() {
+  getString: function(key) {
```
Comment (AiNoKame) [inline app/scripts/modules/payments/components/payment-create.jsx]: Is this method unused?

Comment (hserang) [inline app/scripts/modules/payments/components/payment-create.jsx]: Great catch - started with that and then built the label - let me kill it.

File: app/scripts/modules/payments/components/payment-detail-content.jsx [hunk 2]
```diff
@@ -2,53 +2,74 @@
...
-              </div>
-            </div>
-            <div className="row">
-              <div className="col-sm-9 col-sm-offset-3">
+                <span className="header">
+                  <FormattedMessage
+                    message={this.getIntlMessage('paymentBalanceChanges')}
+                  />
+                </span>
                 <span className="data">{this.props.from_amount} </span>
```
Comment (AiNoKame) [inline app/scripts/modules/payments/components/payment-detail-content.jsx]: Should this amount string be formatted?

File: app/scripts/modules/payments/components/payment-detail-content.jsx [hunk 3]
```diff
@@ -58,49 +79,50 @@ var PaymentDetailContent = React.createClass({
...
         </div>
-        <br />
         <div className="row">
-          <span className="header">Memos: </span>
+          <span className="header">
+            <FormattedMessage message={this.getIntlMessage('paymentMemos')} />
+          </span>
           <span className="data">
             { !_.isEmpty(this.props.memos) ?
               this.props.memos[0].MemoData : 'none'
```
Comment (AiNoKame) [inline app/scripts/modules/payments/components/payment-detail-content.jsx]: Should the 'none' string be localized?

File: gulpfile.js [hunk 4]
```diff
@@ -33,6 +33,9 @@ var merge = require('merge-stream');
 var browserify = require('browserify');
 var del = require('del');
 var reactify = require('reactify');
```
Comment (AiNoKame) [inline gulpfile.js]: This can be deleted

File: app/scripts/modules/payments/components/payment-detail-content.jsx (hunk 2 above)
Comment (hserang) [inline app/scripts/modules/payments/components/payment-detail-content.jsx]: Isn't amount a number? If so, then no. We are not localizing numbers and currency at this point, just the strings.

File: app/scripts/modules/payments/components/payment-detail-content.jsx (hunk 3 above)
Comment (hserang) [inline app/scripts/modules/payments/components/payment-detail-content.jsx]: Great catch! Thank you.

File: gulpfile.js (hunk 4 above)
Comment (hserang) [inline gulpfile.js]: Please clarify - we need ```browserify``` ```del``` and ```reactify```

Comment (AiNoKame) [inline gulpfile.js]: Nevermind! Thanks

**Pull request merged**